### DISCUSSION TITLE
How to pipe stdout into another program when using debugger?

### DISCUSSION THREAD

**Discussion by njakob** [microsoft/vscode-cpptools#7385] [Q&A]
Hello!

I'm trying to pipe the standard output of the debugged application into another program while using gdb through VS code. More concretely, I'm trying to transform the output through `jq`.

This can be achieved directly with gdb:
```sh
$ gdb ./a.out
(gdb) run | jq
```

Is there a way through the `launch.json` configuration file to achieve the same?

**Comment by lingr7** [comment]
I'm trying to redirect the debug console output to .txt or pipe `|` ，but I can't do not.
I try many methos like add args in `launch.json`
```json
   "args": [
                "| tee -a ${workspaceFolder}/info.txt"
            ],
```

```json
   "args": [
                " > ${workspaceFolder}/info.txt"
            ],
```
```json
   "args": [
                " > ",
                "${workspaceFolder}/info.txt"
            ],
```
but all above ,do not work , can not see the info.txt file.
